### ISSUE TITLE
Errors are not being sent to Sentry

### ISSUE THREAD

**Issue by Changaco** [liberapay/liberapay.com#355]
```
ERROR:sentry.errors:Unable to reach Sentry log server: 'module' object has no attribute '_create_default_https_context' (url: https://sentry.changaco.oy.lc/api/store/)
Traceback (most recent call last):
  File "/var/lib/openshift/.../app-root/runtime/dependencies/python/virtenv/lib/python2.7/site-packages/raven/base.py", line 448, in send_remote
    self._send_remote(url=url, data=data, headers=headers)
  File "/var/lib/openshift/.../app-root/runtime/dependencies/python/virtenv/lib/python2.7/site-packages/raven/base.py", line 422, in _send_remote
    return transport.send(data, headers)
  File "/var/lib/openshift/.../app-root/runtime/dependencies/python/virtenv/lib/python2.7/site-packages/raven/transport/base.py", line 164, in send
    response = urllib2.urlopen(req, data, self.timeout).read()
  File "/opt/rh/python27/root/usr/lib64/python2.7/urllib2.py", line 154, in urlopen
    return opener.open(url, data, timeout)
  File "/opt/rh/python27/root/usr/lib64/python2.7/urllib2.py", line 431, in open
    response = self._open(req, data)
  File "/opt/rh/python27/root/usr/lib64/python2.7/urllib2.py", line 449, in _open
    '_open', req)
  File "/opt/rh/python27/root/usr/lib64/python2.7/urllib2.py", line 409, in _call_chain
    result = func(*args)
  File "/opt/rh/python27/root/usr/lib64/python2.7/urllib2.py", line 1258, in https_open
    context=self._context, check_hostname=self._check_hostname)
  File "/opt/rh/python27/root/usr/lib64/python2.7/urllib2.py", line 1183, in do_open
    h = http_class(host, timeout=req.timeout, **http_conn_args)
  File "/opt/rh/python27/root/usr/lib64/python2.7/httplib.py", line 1196, in __init__
    context = ssl._create_default_https_context()
AttributeError: 'module' object has no attribute '_create_default_https_context'
ERROR:sentry.errors:Failed to submit message: "..."
```

**Comment by Changaco** [comment]
It seems this may have been "fixed" when the app crashed two days ago (#354) and was subsequently restarted. That would mean the problem is related to #150.

**Comment by Changaco** [comment]
This hasn't happened again, closing.
